fossunited/fossunited/web_template/home_page___events/home_page___events.html

```diff
@@ -0,0 +1,374 @@
+
+    }
+
+    .event-details{
+
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        gap: 0.5rem;
+        color: var(--gray-800, #383838);
+        font-size: 0.875rem;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 115%; /* 1.00625rem */
+        letter-spacing: -0.035rem;
+    }
+
+    .event-details:hover{
+        text-decoration: none;
+        color: var(--Foundation-FOSS-Mint-foss-mint-500, #08B74F);
+    }
+
+    .events-button{
+
+        border-radius: 0.5rem;
+        background: var(--green-200, #DAF0E1);
+        border: none;
+        padding: 0.8rem 1.5rem;
+        color: var(--green-800, #16794C);
+        font-size: 1rem;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 115%; /* 1.15rem */
+        letter-spacing: -0.02rem;
+    }
+
+    .mobile-only{
+        display: none;
+    }
+
+    @media screen and (max-width: 768px) {
+
+        .events-section{
+            padding: 0rem 2rem 0rem 2rem;
+
+        }
+
+
+        .events-container{
+            max-width: 21.9375rem;
+            gap: 7rem;
+        }
+
+        .events-block{
+            grid-template-columns: 1fr;
+            gap: 2.5rem;
+            width: 100%
+        }
+
+        .flagship-event-card{
+            border-radius: 0.5rem;
+            border: 0.5px solid var(--Gray-200, #EAECF0)
+                }
+
+        .view-events-link{
+            display: none;
+        }
+
+        .mobile-only{
+            display: block;
+        }
+
+    }
+
+
+
+</style>
+
+
+
+<div class="events-section">
+    <div class="events-container">
+
+        <!-- Flagship Events Section -->
+        <div class="events-subsection">
+            <h2>Flagship Events</h2>
+
+            <div class="events-block">
+
+                <a class="flagship-event-card" href="">
+                    <img class="flagship-card-image" src="assets/fossunited/icons/indiafoss-icon.svg" alt="IndiaFOSS">
+                    <h3 class="flagship-card-heading">IndiaFOSS</h3>
+                    <p class="flagship-card-desc">IndiaFOSS is an annual Free and Open Source Software (FOSS) conference,
+                        organised with passion and commitment by the FOSS United community. </p>
+
+                    <p class="flagship-card-cta">More >></p>
+
+                </a>
+
+                <a class="flagship-event-card" href="">
+
+                    <img class="flagship-card-image" src="assets/fossunited/icons/fosshack-icon.svg" alt="IndiaFOSS">
+                    <h3 class="flagship-card-heading">FOSS Hack</h3>
+                    <p class="flagship-card-desc">FOSS Hack is our community-run flagship hackathon to promote Free and Open Source Software by bringing
+                        together students and professionals to build or extend FOSS projects.</p>
+
+                    <p class="flagship-card-cta">More >></p>
+
+                </a>
```

Comment:
Why not have these as fields in template. The content would be easier to change that way.